teslajsonpy/sentry_mode.py

```diff
@@ -43,11 +44,21 @@ def __init__(self, data, controller):
         last_update = self._controller.get_last_update_time(self._id)
         if last_update >= self.__manual_update_time:
             data = self._controller.get_state_params(self._id)
-            self.__sentry_mode = data and data["sentry_mode"]
+            self.__sentry_mode_available = data and data["sentry_mode_available"]
+            if self.__sentry_mode_available:
+                self.__sentry_mode = data["sentry_mode"]
+
+    def is_available(self):
```

Comment:
Change this to `available` so [HA can use it directly](https://developers.home-assistant.io/docs/entity_index#generic-properties).

Reply:
Changed to `available`.